app/models/instructor.rb

```diff
@@ -32,4 +32,20 @@ def self.get_my_tas(instructor_id)
     end
     ta_ids
   end
+
+  def self.get_user_list
+    participants = []
+    Course.where(instructor_id: self.id).find_each do |course|
```

Comment:
It seems that you just simply move the code, without check the functionality.
Your refactor does not work!

Reply:
Sorry about that, we will fix that as soon as possible.